test/performance/latency-controller/README.md

```diff
@@ -1,242 +1,177 @@
-# kubestellar-latency-collector
+# Framework to Benchmark KubeStellar Data-Plane Latencies
 
-> A Kubernetes controller to measure and expose latency metrics for KubeStellar’s downsync and upsync operations.
+<p align="center">
+  <img src="images/Diagram.png" width="700" height="300" title="Benchmarking of KubeStellar  Data-Plane ">
+</p>
 
----
+## Overview
 
-## 🚀 Demo Setup
+The KubeStellar Latency Controller is a comprehensive monitoring solution designed to measure and track end-to-end latencies across multi-cluster workload deployments in KubeStellar environments. It provides detailed metrics collection and observability for the complete workload deployment pipeline from Workload Definition Space (WDS) to Workload Execution Clusters (WECs).
 
-Create a KubeStellar demo environment using Kind:
+## Architecture
```

Comment:
Remove this title "Architecture" and let's include the content below it into the "Overview" section.